packages/ERTP/src/amountMath.js

```diff
@@ -83,6 +83,10 @@ function makeAmountMath(brand, mathHelpersName) {
         return allegedAmount;
       }
       const { brand: allegedBrand, extent } = allegedAmount;
+      assert(
+        allegedBrand !== undefined,
+        details`alleged brand is null. Did you pass an extent rather than an amount?`,
```

Comment:
Can we change the error message to say undefined rather than null?

Reply:
Yup. Good catch.

Reply:
Yes, of course.